packages/babel-helpers/src/helpers.js

```diff
@@ -871,7 +871,7 @@ helpers.taggedTemplateLiteralLoose = helper("7.0.0-beta.0")`
 
 helpers.readOnlyError = helper("7.0.0-beta.0")`
```

Comment:
I didn't update this to avoid inlining the helper if a user is using an older `@babel/runtime`, since the plugins aren't relying on the TypeError anyway.